MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -559,7 +559,7 @@ - (void)showConfirmationAlert:(MSReleaseDetails *)details {
                                          }];
     }
 
-    if ([details.releaseNotes length] > 0) {
+    if ([details.releaseNotes length] > 0 && details.releaseNotesUrl) {
```

Comment:
Yes, it will be a part of release details. ReleaseDetails is a public facing data model, customers might want to get release notes.